flexmeasures/data/schemas/sensors.py

```diff
@@ -222,7 +225,12 @@ def __init__(
             # Insert validation into self.validators so that multiple errors can be stored.
             validator = RepurposeValidatorToIgnoreSensors(_validate)
             self.validators.insert(0, validator)
-        self.to_unit = ur.Quantity(to_unit)
+        if len(to_unit) > 0 and to_unit[0] == "/":
+            self.to_unit = ur.Quantity(to_unit[1:])
```

Comment:
If `to_unit == "/"`, to_unit[1:] -> "" And this would fail:

```
>>> from flexmeasures.utils.unit_utils import ur
>>> ur.Quantity("")
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/home/victor/Work/Seita/flexmeasures/venv-3.10/lib/python3.10/site-packages/pint/facets/plain/quantity.py", line 199, in __new__
    raise ValueError(
ValueError: Expression to parse as PlainQuantity cannot be an empty string.
```